src/include/connection.h

```diff
@@ -417,7 +417,40 @@ struct __wt_connection_impl {
 
 	bool	 mmap;			/* mmap configuration */
 	int page_size;			/* OS page size for mmap alignment */
-	uint32_t verbose;
+
+#define	WT_VERB_API			0x0000000000000001
+#define	WT_VERB_BLOCK			0x0000000000000002
+#define	WT_VERB_CHECKPOINT		0x0000000000000004
+#define	WT_VERB_CHECKPOINT_PROGRESS	0x0000000000000008
+#define	WT_VERB_COMPACT			0x0000000000000010
+#define	WT_VERB_EVICT			0x0000000000000020
+#define	WT_VERB_EVICTSERVER		0x0000000000000040
+#define	WT_VERB_EVICT_STUCK		0x0000000000000080
+#define	WT_VERB_FILEOPS			0x0000000000000100
+#define	WT_VERB_HANDLEOPS		0x0000000000000200
+#define	WT_VERB_LOG			0x0000000000000400
+#define	WT_VERB_LOOKASIDE		0x0000000000000800
+#define	WT_VERB_LSM			0x0000000000001000
+#define	WT_VERB_LSM_MANAGER		0x0000000000002000
+#define	WT_VERB_METADATA		0x0000000000004000
+#define	WT_VERB_MUTEX			0x0000000000008000
+#define	WT_VERB_OVERFLOW		0x0000000000010000
+#define	WT_VERB_READ			0x0000000000020000
+#define	WT_VERB_REBALANCE		0x0000000000040000
+#define	WT_VERB_RECONCILE		0x0000000000080000
+#define	WT_VERB_RECOVERY		0x0000000000100000
+#define	WT_VERB_RECOVERY_PROGRESS	0x0000000000200000
+#define	WT_VERB_SALVAGE			0x0000000000400000
+#define	WT_VERB_SHARED_CACHE		0x0000000000800000
+#define	WT_VERB_SPLIT			0x0000000001000000
+#define	WT_VERB_TEMPORARY		0x0000000002000000
+#define	WT_VERB_THREAD_GROUP		0x0000000004000000
+#define	WT_VERB_TIMESTAMP		0x0000000008000000
+#define	WT_VERB_TRANSACTION		0x0000000010000000
+#define	WT_VERB_VERIFY			0x0000000020000000
+#define	WT_VERB_VERSION			0x0000000040000000
+#define	WT_VERB_WRITE			0x0000000080000000
+	uint64_t verbose;
```

Comment:
You're missing the change in `misc.h` to `WT_VERBOSE_ISSET` to use `FLD64_ISSET`.